src/app/GitCommands/Git/GitModule.cs

```diff
@@ -2085,20 +2090,15 @@ public IEnumerable<string> GetSettings(string setting)
             return ((ConfigFileSettings)LocalConfigFile).GetValues(setting);
         }
 
-        public string GetSetting(string setting) => LocalConfigFile.GetValue(setting);
+        public string GetSetting(string setting) => LocalConfigFile.GetValue(setting) ?? "";
         public T? GetSetting<T>(string setting) where T : struct => LocalConfigFile.GetValue<T>(setting);
 
-        public string GetEffectiveSetting(string setting) => EffectiveConfigFile.GetValue(setting);
+        public string GetEffectiveSetting(string setting, string defaultValue = "") => EffectiveConfigFile.GetValue(setting) ?? defaultValue;
         public T? GetEffectiveSetting<T>(string setting) where T : struct => EffectiveConfigFile.GetValue<T>(setting);
 
-        public SettingsSource GetEffectiveSettingsByPath(string path)
-        {
-            return ((ConfigFileSettings)EffectiveConfigFile).ByPath(path);
-        }
-
         public string? GetGitSetting(string setting, string scopeArg, bool cache = false)
         {
-            GitArgumentBuilder args = new("config") { "--includes", scopeArg, "--get", setting };
+            GitArgumentBuilder args = new("config") { "get", scopeArg, "--includes", setting };
```

Comment:
```suggestion
            GitArgumentBuilder args = new("config") { "--get", scopeArg, "--includes", setting };
```

Reply:
Sorry, I was not aware of this git command line change in v2.46
I had to update. So this PR won't work with <v2.46...

Reply:
Good to have this caught before merging!